apps/website/.vuepress/theme/global-components/CdsDemo.vue

```diff
@@ -65,8 +66,7 @@ export default {
   },
   computed: {
     prismText: function () {
-      const trimmedCode = this.code.trim();
-      return Prism.highlight(trimmedCode, Prism.languages.html, 'html');
+      return Prism.highlight(html(this.code), Prism.languages.html, 'html');
```

Comment:
`this.code` is trimmed in the `build` step when the build creates the json with the code content